### PR TITLE
Fix warnings with Stadia (ABC-393)

### PR DESCRIPTION
**Pull request by thomas-tu**
## Purpose of this PR

**Ticket/Jira #:** https://jira.unity3d.com/browse/ABC-393

References to Stadia are being deprecated from the Editor since 2023.1+ as the platform is now down.
This PR removes these so we the Editor stops throwing warnings in the Console.

## Testing

**Functional Testing status:**

Manual test by installing the package with the changes and verifying that we don't get warnings.

**Performance Testing status:** N/A

## Overall Product Risks

**Complexity:** Minimal

**Halo Effect:** Minimal

## Documentation & UX Writing

| User facing text to review |  | Details |
| :--- | :--- | :--- |
| User interface | N | |
| Public API docs | N | |
| Changelog | Y |  |

| Documentation halo effect |  | Jira link |
| :--- | :--- | :--- |
| User manual | Y |  |
| Other docs | N |  |

### CODE REVIEW

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2023.1.0b6 (Windows only)

File: com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs [hunk 1]
```diff
@@ -66,7 +66,7 @@ public static void OnPostProcessBuild(BuildTarget target, string pathToBuiltProj
 
         public static bool TargetIsSupported(BuildTarget target)
         {
-            return target == BuildTarget.StandaloneOSX || target == BuildTarget.StandaloneWindows64 || target == BuildTarget.StandaloneLinux64 || target == BuildTarget.Stadia;
+            return target == BuildTarget.StandaloneOSX || target == BuildTarget.StandaloneWindows64 || target == BuildTarget.StandaloneLinux64;
```
Comment (RoseHirigoyen) [inline com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs:69]: Just to clarify, this just prevents the user from building on stadia?

**Review by RoseHirigoyen — APPROVED** [review]

**Comment by lucillecaillaud** [comment]
Should we do something specific to keep supporting it before 2023.1? Or we don't care?

**Comment by cguer** [comment]
> Should we do something specific to keep supporting it before 2023.1? Or we don't care?

The [Google Stadia](https://stadia.google.com/) service was shut down completely in January. From what I understand, even if a user were to build a game targeting Stadia there is just no way to play them or host them anywhere. (so I vote that we don't care 😂 )

**Comment by thomas-tu** [comment]
I had in mind exactly what Christophe said when I made this change :D

File: com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs [hunk 2]
```diff
@@ -16,7 +16,7 @@ namespace UnityEditor.Formats.Alembic.Importer
 {
     static class AlembicBuildPostProcess
     {
-        internal const string kUnsupportedTarget = "Alembic only supports the following build targets: Windows 64-bit, MacOS X, Linux 64-bit or Stadia.";
+        internal const string kUnsupportedTarget = "Alembic only supports the following build targets: Windows 64-bit, MacOS X or Linux 64-bit.";
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs:19]:
```suggestion
        internal const string kUnsupportedTarget = "Alembic only supports the following build targets: Windows 64-bit, macOS X, and Linux 64-bit.";
```

File: com.unity.formats.alembic/package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "com.unity.modules.unityanalytics": "1.0.0",
     "com.unity.modules.cloth": "1.0.0"
   },
-  "description": "Import and export Alembic files (.abc), record and stream animations in the Alembic format directly in Unity.\nNOTE: Supported build platforms: only Windows 64, MacOS X, Linux 64, and Stadia.\n\nThe Alembic format is commonly used in animation to transfer facial, cloth, and other simulation between applications in the form of baked data.",
+  "description": "Import and export Alembic files (.abc), record and stream animations in the Alembic format directly in Unity.\nNOTE: Supported build platforms: only Windows 64, MacOS X and Linux 64.\n\nThe Alembic format is commonly used in animation to transfer facial, cloth, and other simulation between applications in the form of baked data.",
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/package.json:7]:
```suggestion
  "description": "Import and export Alembic files (.abc), record and stream animations in the Alembic format directly in Unity.\nNOTE: Supported build platforms: only Windows 64, macOS X, and Linux 64.\n\nThe Alembic format is commonly used in animation to transfer facial, cloth, and other simulation between applications in the form of baked data.",
```

File: com.unity.formats.alembic/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Removed
+- Removed support of Stadia.
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/CHANGELOG.md:9]:
```suggestion
- Removed support of Stadia as a build target.
```

**Review by sebastienduverne — APPROVED** [review]
Minor things.

**Pull request merged**